server/config/routes.js

```diff
@@ -20,6 +20,13 @@ module.exports.routes = {
 
   "get /api/cache/refresh-cache": "cache/refresh-cache",
 
+  "post /api/forms/create-form": "forms/create-form",
```

Comment:
mayb arrange this a bit neater (use comments, group gets and posts together etc.) to help solve merge conflicts in the future

Reply:
yea this line is actually a merge conflict. it shouldn't be there